jina/peapods/pods/__init__.py

```diff
@@ -613,3 +531,64 @@ def dump(self, path, shards, timeout):
                     DumpMessage(path=path, shards=shards),
                     timeout=timeout,
                 )
+
+    @staticmethod
```

Comment:
If u separate the code for Pod and CompoundPod, this should be kept as set pea args